enhancements/sig-architecture/32-extensiblescheduling/32-extensiblescheduling.md

```diff
@@ -0,0 +1,462 @@
+# placement extensible scheduling
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/website/)
+
+## Summary
+
+The proposed work provides an API to represent the score of the managed cluster, to support placement extensible scheduling.
+
+## Motivation
+
+When implementing placement resource based scheduling, we find in some cases, the prioritizer needs extra data (more than the default value provided by `ManagedCluster`) to calculate the score of the managed cluster. For example, there is a requirement to schedule based on resource monitoring data from the cluster.
+
+So we want a more extensible way to support scheduling based on customized scores.
+
+### Goals
+- Design a new API(CRD) to contain the customized scores for each managed cluster.
+- Let placement prioritizer support rating clusters with the customized scores provided by the new API(CRD).
+
+### Non-Goals
+- How to maintain the lifecycle (create/update/delete) of the CRs.
+- Placement filters will not support the new API(CRD).
```

Comment:
the details from your answer https://github.com/open-cluster-management-io/enhancements/pull/33#discussion_r755819357 are very good.  Can you add them here:

> Put filter out of scope is because:
Most of the cases we know are that users want to use customized values to prioritize clusters.
For filters, adding labels to filter is easier than supporting customized values to filter.

Reply:
OK, will do it.